src/lib/connectors/shared/remote-debugging-connector.ts

```diff
@@ -332,12 +325,16 @@ export class Connector implements IConnector {
             };
         }
 
+        if (data.element && data.element.nodeName.toLowerCase() === 'link' && data.element.getAttribute('rel') === 'icon') {
```

Comment:
The `data.element.getAttribute('rel') === 'icon'` check will fail for something like [`REL="SHORTCUT ICON"`](https://mathiasbynens.be/notes/rel-shortcut-icon).